HelloWorld-CMake/README.md

```diff
@@ -20,7 +20,7 @@ If `swiftc` is not in your path, you will need to add `-DCMAKE_Swift_COMPILER=`
 with the path to swiftc.
 
 ```sh
-cmake -G Ninja -B build -DCMAKE_BUILD_TYPE=RelWithDebInfo -DBUILD_TESTING=YES
+cmake -G Ninja -B build -DCMAKE_BUILD_TYPE=RelWithDebInfo -DBUILD_TESTING=YES -D CMAKE_Swift_FLAGS="-sdk <Path to SDK> -I <Path to SDK>/usr/lib/swift -L <Path to SDK>/usr/lib/swift/<os>"
```

Comment:
This is no longer copy paste able.  I think that it might be useful to have some exposition setting up environment variables on windows and Linux and macOS respectively and then explain how to reference those